.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Coverage Report
+
+on:
+  push:
+    branches:
+      - playground
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4.1.0
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests with coverage
+        run: npm test
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

Comment:
Is it possible to use v5 instead of v3?
https://github.com/marketplace/actions/codecov